### DISCUSSION TITLE
final grade

### DISCUSSION THREAD

**Discussion by billbiancheng** [PaddlePaddle/PaddleSpatial#129] [BaiduKDDCup2022] [answered]
Dear Organizer,
      Do online grades represent final grades? Or there are also new test sets for final evaluation。Thanks so much.

**Comment by ramber1836** [comment, marked as answer]
No, it's not.
FYI, 
![image](https://user-images.githubusercontent.com/19561583/171312934-645128b4-ecc5-4749-ba5b-483b096a7a05.png)
